config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
+  get 'islands/index'
+  get 'islands/show'
+  get 'islands/new'
+  get 'islands/create'
```

Comment:
Not necessary when we add resources :islands.

Reply:
All fixed with the unnecessary routes! Thanks for the comment.